index.html

```diff
@@ -232,12 +236,18 @@ <h2 class="text-center">Get in touch with us</h2>
                 <input id="tel" name="tel" type="tel">
 
                 <label for="message">Message</label>
-                <textarea name="message"></textarea>
+                <textarea name="message" id="message"></textarea>
 
                 <button type="submit" class="submit-btn">Submit</button>
             </form>
         </section>
     </main>
+    
+    <footer>
```

Comment:
the content that you've added for the footer is great, ✨ I may add some more details about the agency itself such as address, tell and link logo's on where else users can find the agency